src/jennifer/adapter/mysql/command_interface.cr

```diff
@@ -17,7 +17,8 @@ module Jennifer
 
       def generate_schema
         options = ["-u", config.user, "--no-data", "-h", config.host, "--skip-lock-tables", config.db]
-        options += ["--password=#{config.password}"] if !config.password.empty?
+        options += ["--port=#{config.port}"]
```

Comment:
`port` configuration property may not be set. Let's make it optional same like `password` below

Reply:
Updated,
I also changed the condition `if !config.password.empty?`  to `unless config.password.empty?`